test/inactive-blog-filter.test.js

```diff
@@ -129,54 +118,57 @@ describe('Testing of update()', () => {
   }
 
   it('should not redlist active blogs', async () => {
-    return inactiveFilter
-      .update(pathToActiveFeeds, pathToMockRedList, mockFeedParser)
-      .then(data => {
-        expect(data.length).toEqual(0);
-      })
-      .catch(err => {
-        deleteMockRedlist();
-        log.error(err.message);
-        throw err;
-      });
+    try {
+      const data = await inactiveFilter.update(
+        pathToActiveFeeds,
+        pathToMockRedList,
+        mockFeedParser
+      );
+      expect(data.length).toEqual(0);
+    } catch (err) {
+      deleteMockRedlist();
+      log.error(err.message);
+      throw err;
+    }
   });
 
   it('should redlist inactive blogs', async () => {
     const inactiveRedlist = mockFeeds.filter(item => {
       return item.status === 'inactive';
     });
-    return inactiveFilter
-      .update(pathToInactiveFeeds, pathToMockRedList, mockFeedParser)
-      .then(data => {
-        expect(data.length).toEqual(inactiveRedlist.length);
-        for (let i = 0; i < inactiveRedlist.length; i += 1) {
-          expect(data[i].url).toBe(inactiveRedlist[i].url);
-        }
-      })
-      .catch(err => {
-        deleteMockRedlist();
-        log.error(err.message);
-        throw err;
-      });
+    try {
+      const data = await inactiveFilter.update(
+        pathToInactiveFeeds,
+        pathToMockRedList,
+        mockFeedParser
+      );
+      expect(data.length).toEqual(inactiveRedlist.length);
+      for (let i = 0; i < inactiveRedlist.length; i += 1) {
+        expect(data[i].url).toBe(inactiveRedlist[i].url);
+      }
+      return data;
+    } catch (err) {
+      deleteMockRedlist();
+      log.error(err.message);
```

Comment:
I think in unit tests, it's better to just let things throw, and not bother with catching, unless we want to test something in the error handling path.  The test harness will blow up if there's an exception, and print it, so we don't have to.